Documentation/HowTo/custom-markdown-metadata-values.md

```diff
@@ -1,6 +1,6 @@
 # How to express custom metadata values using Markdown
 
-Publish enables each website to define its own site-specific item metadata, through its `Website.ItemMetadata` type. When adding items using Markdown, those values can then be expressed by adding a metadata header at the top of a file.
+Publish enables each website to define its own site-specific item metadata, through its `Website.ItemMetadata` type. When adding items using Markdown, those values can then be expressed by adding a metadata header at the top of a file (commonly known as *front matter*).
```

Comment:
I wouldn't go so far as to say that "front matter" is "commonly known" (I only heard about it for the very first time the other day myself). How about phrasing this sentence as "within other tools referred to as *front matter*"?

Reply:
that sounds good. let me do the change 😉